Containers/recording/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM alpine:3.18.0
+
+COPY --chmod=775 start.sh /usr/bin/start.sh
+COPY --chmod=664 recording.conf /etc/recording.conf
+
+RUN set -ex; \
+    apk add --no-cache \
+        ca-certificates \
+        tzdata \
+        bash \
+        python3 \
+        py3-pip \
+        ffmpeg \
+        libpulse \
+        bind-tools \
+        netcat-openbsd \
+        git \
+        wget \
+        shadow; \
+    # xvfb firefox chromium chromium-chromedriver?
+    # apk add --no-cache geckodriver --repository http://dl-cdn.alpinelinux.org/alpine/edge/testing; \
```

Comment:
not sure if they are required, since building the container / launching it works without it, no Idea if they are required while recording

Reply:
I guess we'll find out. Good to leave it in as a comment for now :)